src/SFA.DAS.PensionsRegulatorApi.UnitTests/Controllers/PayeAndAorn/Given_No_Results_Are_Returned.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using NSubstitute;
+using NUnit.Framework;
+using PensionsRegulatorApi.Application.Queries;
+using PensionsRegulatorApi.Controllers;
+using PensionsRegulatorApi.Domain;
+
+namespace SFA.DAS.PensionsRegulatorApi.UnitTests.Controllers.PayeAndAorn
+{
+    [TestFixture]
+    [ExcludeFromCodeCoverage]
+    public class Given_No_Results_Are_Returned
```

Comment:
Naming same as previous comment, less technical names would be good.

Reply:
Done a rewrite of the controller tests now, see what you think.